processes/dags/pubmed_migration.py

```diff
@@ -22,6 +22,7 @@
 # Setting up MongoDB hook to mlab server
 mongodb_hook = MongoHook('mongo_default')
 ftp_conn_id = "pubmed_ftp"
+CARDIOVASCULAR = "blood/heart and lymphatics"
```

Comment:
Does it work to have blood/heart, or should those be listed separately?

Reply:
i basically gathered all subjects i found in a subset and this was the closest thing to cardiovascular I found. I'm not exactly sure how they pick their subject names, but this was  found in a significant chunk of them. I might redo this later and try and make it more sophisticated, but i just wanted to get something out for joey first.